frontend/device/sdl/device.lua

```diff
@@ -59,6 +59,16 @@ function Device:init()
     Generic.init(self)
 end
 
+function Device:setTime(hour, min)
```

Comment:
Won't this crash if there isn't some kind of dummy function in Device that returns false? (should be easy to test by removing it from SDL)